activerecord/test/cases/invertible_migration_test.rb

```diff
@@ -157,8 +157,10 @@ def change
 
```

Comment:
In these migration tests, we should check table is created (or dropped), so can not change.